Update install-and-configure-workflow-for-sharepoint-server.md (#3708)
MicrosoftDocs/OfficeDocs-SharePoint: SharePoint/SharePointServer/governance/install-and-configure-workflow-for-sharepoint-server.md

   > [!NOTE]
   >There is no need to delete the existing Workflow Service Application Proxy, and there is no need to re-register SPWorkflowService.

   > [!NOTE]

>    >In some situations you might get a message that the Certificate Generation Key is invalid. In this case, please follow the instructions in the following article to resest the Certificate Generation Key: [Reset Certificate Generation Key for WorkFlowManager and ServiceBus](https://learn.microsoft.com/archive/blogs/biztalknotes/reset-certificate-generation-key-for-workflowmanager-and-servicebus)

Reviewer: ```suggestion
   > In some situations you might get a message that the Certificate Generation Key is invalid. If that happens, you should follow the instructions in the following article to reset the Certificate Generation Key: [Reset Certificate Generation Key for WorkFlowManager and ServiceBus](/archive/blogs/biztalknotes/reset-certificate-generation-key-for-workflowmanager-and-servicebus).
```
